thinc/types.py

```diff
@@ -443,11 +467,11 @@ def __get_validators__(cls):
             yield validator
 
 
-f2d1 = TypeVar("f2d1", bound=int)
```

Comment:
I removed the binding here because it doesn't appear like typing will deal with multiple types, and it says that bindings shouldn't be Union types. Without removing the binding you can't use Floats2d to represent an unknown dimension. I guess if we really wanted to bind to int we could use something like -1 to represent a None dimension?

Reply:
reverted this for now since we're not doing shape checking